### PR TITLE
Add docs for dbt-greenplum adapter

### PR DESCRIPTION
**Pull request by markporoshin**
## Description & motivation
I implemented dbt-adapter for Greenplum, so would like to add documentation for it.

## Checklist
- [x] The pages have been added to `website/sidebars.js`
- [x] The new pages have a unique filenames

### CODE REVIEW

File: website/docs/reference/resource-configs/greenplum-configs.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    
+Table in greenplum have a few powerful optimizations to improve query perfomance:
+ 
+ - distribution
+ - column orientation
+ - compression
+ - appendonly toggle
+ - partitions
+ 
+Supplying these values as model-level configurations apply the corresponding settings in the generated `CREATE TABLE`(except partitions). Note that these settings will have no effect for models set to `view`.
```
Comment (dataders) [inline website/docs/reference/resource-configs/greenplum-configs.md:16]: perhaps you could throw an error message if these configs are set for a view model?

Comment (dataders) [reply website/docs/reference/resource-configs/greenplum-configs.md:16]: also, why does `partitions` not apply to a `CREATE TABLE` statement?

File: website/docs/reference/resource-configs/greenplum-configs.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+---
+
+## Performance Optimizations
+    
+Table in greenplum have a few powerful optimizations to improve query perfomance:
+ 
+ - distribution
+ - column orientation
+ - compression
+ - appendonly toggle
```
Comment (dataders) [inline website/docs/reference/resource-configs/greenplum-configs.md]:
```suggestion
 - `appendonly` toggle
```

File: website/docs/reference/resource-configs/greenplum-configs.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+ - partitions
+ 
+Supplying these values as model-level configurations apply the corresponding settings in the generated `CREATE TABLE`(except partitions). Note that these settings will have no effect for models set to `view`.
+
+### Distribution
+
+In greenplum you have opportunity to choose [distribution key](https://gpdb.docs.pivotal.io/6-4/admin_guide/distribution.html), that will be used to sort data by segments. Join-s by this key will be more effective.
```
Comment (dataders) [inline website/docs/reference/resource-configs/greenplum-configs.md]:
```suggestion
In Greenplum, you can choose a [distribution key](https://gpdb.docs.pivotal.io/6-4/admin_guide/distribution.html), that will be used to sort data by segments. Joining on the partition will become more performant after specifying distribution.
```

File: website/docs/reference/resource-configs/greenplum-configs.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+ - appendonly toggle
+ - partitions
+ 
+Supplying these values as model-level configurations apply the corresponding settings in the generated `CREATE TABLE`(except partitions). Note that these settings will have no effect for models set to `view`.
+
+### Distribution
+
+In greenplum you have opportunity to choose [distribution key](https://gpdb.docs.pivotal.io/6-4/admin_guide/distribution.html), that will be used to sort data by segments. Join-s by this key will be more effective. 
+
+By default dbt-greenplum distribute data `RANDOMLY`. To implement distribution key you need to specify `distributed_by` parameter in model config:
```
Comment (dataders) [inline website/docs/reference/resource-configs/greenplum-configs.md]:
```suggestion
By default dbt-greenplum distributes data `RANDOMLY`. To implement a distribution key you need to specify the `distributed_by` parameter in model's config:
```

File: website/docs/reference/resource-configs/greenplum-configs.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+        distributed_by='<field_name>'
+        ...
+    )
+}}
+
+
+select ...
+```
+
+Also you can choose `DISTRIBUTED REPLICATED` option:
```
Comment (dataders) [inline website/docs/reference/resource-configs/greenplum-configs.md:37]: what does this do? replicates all the rows to every node?

File: website/docs/reference/resource-configs/greenplum-configs.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
+---
+title: "Greenplum configurations"
+id: "greenplum-configs"
+---
+
+## Performance Optimizations
+    
+Table in greenplum have a few powerful optimizations to improve query perfomance:
```
Comment (dataders) [inline website/docs/reference/resource-configs/greenplum-configs.md]:
```suggestion
Tables in Greenplum have powerful optimization configurations to improve query performance:
```

File: website/docs/reference/resource-configs/greenplum-configs.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+ 
+ - distribution
+ - column orientation
+ - compression
+ - appendonly toggle
+ - partitions
```
Comment (dataders) [inline website/docs/reference/resource-configs/greenplum-configs.md]: you might be able to make these bullet points link to the sections below with 
```suggestion
 - [distribution](### Distribution)
 - [column orientation](### Column orientation)
 - [compression](###Compression)
 - appendonly toggle
 - partitions
```

File: website/docs/reference/warehouse-profiles/greenplum-profile.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
+---
+title: "Greenplum Profile"
+---
+
+## Overview of dbt-postgres
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/greenplum-profile.md]:
```suggestion
## Overview of dbt-greenplum
```

File: website/docs/reference/warehouse-profiles/greenplum-profile.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+```bash
+$ pip install dbt-greenplum
+```
+For further (and more likely up-to-date) info, see the [README](https://github.com/markporoshin/dbt-greenplum#README.md)
+
+
+## Profile Configuration
+
+Postgres targets should be set up using the following configuration in your `profiles.yml` file.
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/greenplum-profile.md]:
```suggestion
Greenplum targets should be set up using the following configuration in your `profiles.yml` file.
```

might also be worth adding the default values of these parameters?

**Review by dataders — CHANGES_REQUESTED** [review]
Nice to have: turn config options into tabs to make the document shorter see [this section's markdown](https://docs.getdbt.com/reference/warehouse-profiles/mssql-profile) for an example

File: website/docs/reference/resource-configs/greenplum-configs.md (hunk 1 above)
Comment (markporoshin) [reply website/docs/reference/resource-configs/greenplum-configs.md:16]: yes, my bad, I mean `create table ... as select ...`

File: website/docs/reference/resource-configs/greenplum-configs.md (hunk 5 above)
Comment (markporoshin) [reply website/docs/reference/resource-configs/greenplum-configs.md:37]: "DISTRIBUTED REPLICATED replicates all rows in the table to all Greenplum Database segments." from [documentation](https://gpdb.docs.pivotal.io/6-9/ref_guide/sql_commands/CREATE_TABLE_AS.html)
So you are right

File: website/docs/reference/resource-configs/greenplum-configs.md (hunk 1 above)
Comment (markporoshin) [reply website/docs/reference/resource-configs/greenplum-configs.md:16]: "perhaps you could throw an error message if these configs are set for a view model?" I created an [issue](https://github.com/markporoshin/dbt-greenplum/issues/9).

**Review by dataders — APPROVED** [review]
I think this is ready to go!

**Pull request merged**